src/components/course/course-header/CoursePreview.jsx

```diff
@@ -40,7 +41,11 @@ const CoursePreview = ({ previewImage, previewVideoURL }) => {
               <Image src={previewImage} className="video-thumb" alt="" />
               <div className="video-trigger-cta btn btn-inverse-primary">
                 <PlayCircleFilled className="mr-1" />
-                Play Video
+                <FormattedMessage
+                  id="enterprise.course.about.page.video.play.button.text"
+                  defaultMessage="Play Video"
+                  description="Play Video button text"
```

Comment:
Can we please add little bit more context? May be `Play video button text. Course introduction video will played when this button will be clicked`